website/docs/r/ingress_v1.html.markdown

```diff
@@ -306,6 +305,13 @@ The following arguments are supported:
 * `ip` -  IP is set for load-balancer ingress points that are IP based (typically GCE or OpenStack load-balancers).
 * `hostname` - Hostname is set for load-balancer ingress points that are DNS based (typically AWS load-balancers).
 
+## Timeouts
+
+The following [Timeout](https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/retries-and-customizable-timeouts) 
+configuration options are available for the `kubernetes_ingress_v1` resource:
```

Comment:
Hi @Tensho,

I think this link is more for the provider developers. Let's use this one: https://developer.hashicorp.com/terraform/language/resources/syntax#operation-timeouts

Could you please make this sentence in one line?

Thanks.

Reply:
Done